pkg/codegen/python/gen.go

```diff
@@ -491,7 +492,14 @@ func (mod *modContext) genResource(res *schema.Resource) (string, error) {
 		}
 
 		// And add it to the dictionary.
-		arg := pname
+		arg = pname
+
+		if prop.ConstValue != nil {
+			arg, err = getConstValue(prop.ConstValue)
```

Comment:
Ah, guessing you made that change to avoid declaring the variables here. I'd suggest adding `var err error` after line 468 instead.